contrib/completion/bash/docker

```diff
@@ -4249,6 +4254,14 @@ _docker_manifest_rm() {
 	esac
 }
 
```

Comment:
Please insert
```shell script
_docker_manifest_list() {
  _docker_manifest_ls
}
```
here so that the options for `list` are also completed.

Reply:
done